HW_1/task_08.py

```diff
@@ -0,0 +1,13 @@
+# Требуется определить, можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+# *Пример:*
+
+# 3 2 4 -> yes
+# 3 2 1 -> no
+
+n = int(input("Введите длинну шоколадки: "))
+m = int(input("Введите ширину шоколадки: "))
+k = int(input("Сколько долек вы хотите отломить?: "))
+
+print('yes' if n * m > k and k % n == 0 or k % m == 0 else 'no')
```

Comment:
Условия построены неверно, тут нужны скобки.
```suggestion
print('yes' if n * m > k and (k % n == 0 or k % m == 0) else 'no')
```
Кроме того, первое условие можно расширить по нижней границе допустимых значений переменной `k`